spec/models/asset_spec.rb

```diff
@@ -3,7 +3,295 @@
+      expect(asset.raw_footage_title.include?("Test raw_footage_title")).to be true
+    end
+  end
+
+  context "promo_title" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has promo_title" do
+      asset.promo_title = ["Test promo_title"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasPromoTitle/)
+      expect(asset.promo_title.include?("Test promo_title")).to be true
+    end
   end
+
+  context "clip_title" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has clip_title" do
+      asset.clip_title = ["Test clip_title"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasClipTitle/)
+      expect(asset.clip_title.include?("Test clip_title")).to be true
+    end
+  end
+
+  context "program_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has program_description" do
+      asset.program_description = ["Test program_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasProgramDescription/)
+      expect(asset.program_description.include?("Test program_description")).to be true
+    end
+  end
+
+  context "episode_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has episode_description" do
+      asset.episode_description = ["Test episode_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasEpisodeDescription/)
+      expect(asset.episode_description.include?("Test episode_description")).to be true
+    end
+  end
+
+  context "segment_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has segment_description" do
+      asset.segment_description = ["Test segment_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasSegmentDescription/)
+      expect(asset.segment_description.include?("Test segment_description")).to be true
+    end
+  end
+
+  context "raw_footage_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has raw_footage_description" do
+      asset.raw_footage_description = ["Test raw_footage_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasRawFootageDescription/)
+      expect(asset.raw_footage_description.include?("Test raw_footage_description")).to be true
+    end
+  end
+
+  context "promo_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has promo_description" do
+      asset.promo_description = ["Test promo_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasPromoDescription/)
+      expect(asset.promo_description.include?("Test promo_description")).to be true
+    end
+  end
+
+  context "clip_description" do
+    let(:asset) { FactoryBot.build(:asset) }
+    it "has clip_description" do
+      asset.clip_description = ["Test clip_description"]
+      expect(asset.resource.dump(:ttl)).to match(/pbcore.org#hasClipDescription/)
+      expect(asset.clip_description.include?("Test clip_description")).to be true
+    end
+  end
+
+
```

Comment:
The factories used in the model specs can be defined once at the top-level `describe`, e.g. `let(:asset) { FactoryBot.build(:asset) }`, rather than per context.